source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,59 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Enhanced thermal configuration
+===============================
+
+Better thermal control and performance can be achieved by providing platform
+specific configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of :abbr:`Intel® Dynamic Platform and Thermal Framework (`Intel DPTF)`
+technology, and convert to the thermal_conf.xml configuration format used
+by thermald. It's a closed-source project, and unable to be packaged as bundle
+in Clear Linux OS, so we need to follow below steps to generate configuration.
+
+Intel DPTF requires BIOS support, it's typically used by laptop.
+The first step is to make sure your machine's BIOS has DPTF feature
+and is enabled.
+
+Then, check thermald version is equal to or above version 1.9.1:
+
+.. code:: bash
+
+   thermald --version
+
+If its version is not up-to-date, please upgrade to latest Clear Linux version.
+
+Next step, generate thermal configuration as below:
+
+.. code:: bash
+
+   git clone https://github.com/intel/dptfxtract.git
+   cd dptfxtract
+   sudo acpidump > acpi.out
```

Comment:
Can you include a step to ensure the bundle that contains these tools (acpica-unix2) is installed (i.e. `sudo swupd bundle-add acpica-unix2`

Reply:
fixed in [0abdc54](https://github.com/clearlinux/clear-linux-documentation/pull/961/commits/0abdc549c8f9f41c68bca48e6b623365ea6604d2)